release.config.cjs

```diff
@@ -0,0 +1,83 @@
+const transform = require("./release.utils.cjs");
+module.exports = {
+    branches: [
+        "+([0-9])?(.{+([0-9]),x}).x",
+        "master",
+        "next",
+        "next-major",
+        {
+            name: "beta",
+            prerelease: true,
+        },
+        {
+            name: "alpha",
+            prerelease: true,
+        },
+        {
+            name: "development",
+            prerelease: true,
+        },
+    ],
+    repositoryUrl: "git@github.com:deriv-com/deriv-utils.git",
+    plugins: [
+        [
+            "@semantic-release/commit-analyzer",
+            {
+                releaseRules: [
+                    {
+                        type: "feat",
+                        release: "minor",
```

Comment:
Is it a good idea to release a minor version for each feature 🤔 
since we are using feat for each function we add I guess it's not a good idea to release a minor version for it 🤔 
what do u think?

Reply:
Yea for now we can use `patch` for `feat`, in the event we wanna bump it up to minor version we can always change it